style.css

```diff
@@ -454,3 +566,161 @@ textarea {
 .btn3:hover {
   background-color: #fff;
 }
+
+@media screen and (min-width: 768px) {
+  .hamburger {
+    display: none;
+  }
+
+  .container {
+    flex: 1;
+    justify-content: center;
+    align-items: center;
+    align-content: center;
+  }
+
+  .container .head {
+    padding: 0;
+    padding-top: 130px;
+    margin: 0;
+  }
+
+  .container .pr {
+    width: 550px;
+    height: 130px;
+    padding-left: 200px;
+  }
+
```

Comment:
- [ ] As required, you this `container` section should have its child elements centred, And this issues comes from the `width: 550px` you gave to the `.container .pr{}`. It would be nice if you remove that width and just enlarge all your elements to have the same width, centred but aligned to left.
- With `width: 550px`
   
![image](https://user-images.githubusercontent.com/80612925/172898326-b36ea21a-b3bd-4ce5-9f6a-afeb52b121c7.png)

- Without `width`
![image](https://user-images.githubusercontent.com/80612925/172898572-e78cb503-a30a-496b-b2e5-a6f4c16f27d6.png)